src/nrniv/impedanc.cpp

```diff
@@ -498,44 +322,41 @@ void Imp::setmat1() {
 }
 
 void Imp::LUDecomp() {
-	int i, ip;
 	NrnThread* _nt = nrn_threads;
-	int i1, i2, i3;
-	i1 = 0;
-	i2 = i1 + _nt->ncell;
-	i3 = _nt->end;
-	for (i=i3-1; i >= i2; --i) {
-		ip = _nt->_v_parent[i]->v_node_index;
+    int i1 = 0;
+	int i2 = i1 + _nt->ncell;
```

Comment:
very nitpick: could you fix this indentation?

Reply:
tab vs space :(